src/components/POLine/POLineDetails/POLineDetailsForm.js

```diff
@@ -62,7 +62,7 @@ class POLineDetailsForm extends Component {
             />
           </Col>
           <Col xs={6}>
-            <FieldOrderFormat />
+            <FieldOrderFormat {...this.props} />
```

Comment:
It's better to specify props separately, so we know what is actually used inside. You even specified proptypes only for 2 props